modules/gitops-enabling-dex.adoc

```diff
@@ -0,0 +1,48 @@
+// Module is included in the following assemblies:
+//
+// * 
+
+[id="enabling-dex_{context}"]
+= Enabling Dex
+
+Argo CD embeds and bundles Dex as part of its installation. Dex is an identity service that uses OpenID Connect to drive authentication for other apps.
+
+.Procedure
+
+. Enable Dex by updating the Subscription resource for the OpenShift GitOps Operator.
++
+[source,yaml]
+----
+spec:
+  config:
+    env:
+    - name: DISABLE_DEX
+      Value: "false"
+----
++
+This update causes the `argocd-cluster-dex-server` instance to run.
+
+. To enable login with OpenShift, update the `argo-cd` Custom Resource by adding the following field: 
++
+[source,yaml]
+----
+spec:
+  dex:
+    openShiftOAuth: true
+----
+    
+. Enable role-based access control (RBAC) on `argo-cd` by modifying the following fields:
++
+[source,yaml]
+----
+spec:
+  dex:
+    openShiftOAuth: true
+  rbac:
+    defaultPolicy: 'role:readonly' 
+    policy: |
+      g, system:cluster-admins, role:admin 
+    scopes: '[groups]' 
+----
++
+Learn more about Argo CD RBAC configurations in link:https://argoproj.github.io/argo-cd/operator-manual/rbac/[upstream Argo CD documentation].
```

Comment:
We will need Rick's ack for this link. Alternatively drop the link. Folks can google for Argo CD documentation. Also, considering the link is part of additional resources that comes up just after this module, maybe we can drop this link.
Drop the word 'upstream', add article 'the' after 'in'
...configurations in the link:https://argoproj.github.io/argo-cd/operator-manual/rbac/[Argo CD documentation].